### PR TITLE
auth: update EOL matrix with 4.7.0 release

### PR DESCRIPTION
**Pull request by Habbie**
### Checklist

I have:
- [ ] read the [CONTRIBUTING.md](https://github.com/PowerDNS/pdns/blob/master/CONTRIBUTING.md) document
- [ ] compiled this code
- [ ] tested this code
- [ ] included documentation (including possible behaviour changes)
- [ ] documented the code
- [ ] added or modified regression test(s)
- [ ] added or modified unit test(s)
- [ ]  checked that this code was merged to master

### CODE REVIEW

File: docs/appendices/EOL.rst [hunk 1]
```diff
@@ -28,16 +28,16 @@ such a user, these EOL statements do not apply to you.
      - End of Life
    * - 4.6
```
[the remark this answers is not in the document]
Comment (Habbie) [reply docs/appendices/EOL.rst:29]: good point!

**Review by aerique — APPROVED** [review]

File: docs/appendices/EOL.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (Habbie) [reply docs/appendices/EOL.rst:33]: amended, please check

[the remark this answers is not in the document]
Comment (Habbie) [reply docs/appendices/EOL.rst:33]: I did it wrong

[the remark this answers is not in the document]
Comment (Habbie) [reply docs/appendices/EOL.rst:33]: amended again

**Comment by Habbie** [comment]
Added a secpoll update.

**Pull request merged**